frontend/src/metabase/query_builder/components/view/View.jsx

```diff
@@ -356,7 +356,31 @@ class View extends Component {
           updateQuestion={this.props.updateQuestion}
           className={CS.flexNoShrink}
         />
-        <ViewFooter {...this.props} className={CS.flexNoShrink} />
+        <ViewFooter
+          question={this.props.question}
+          result={this.props.result}
+          isShowingChartTypeSidebar={this.props.isShowingChartTypeSidebar}
+          isShowingChartSettingsSidebar={
+            this.props.isShowingChartSettingsSidebar
+          }
+          isShowingRawTable={this.props.isShowingRawTable}
+          onOpenChartType={this.props.onOpenChartType}
+          onOpenModal={this.props.onOpenModal}
+          onCloseChartType={this.props.onCloseChartType}
+          onOpenChartSettings={this.props.onOpenChartSettings}
+          onCloseChartSettings={this.props.onCloseChartSettings}
+          setUIControls={this.props.setUIControls}
+          isObjectDetail={this.props.isObjectDetail}
+          questionAlerts={this.props.questionAlerts}
+          visualizationSettings={this.props.visualizationSettings}
+          canManageSubscriptions={this.props.canManageSubscriptions}
+          isVisualized={this.props.isVisualized}
+          isTimeseries={this.props.isTimeseries}
+          isShowingTimelineSidebar={this.props.isShowingTimelineSidebar}
+          onOpenTimelines={this.props.onOpenTimelines}
+          onCloseTimelines={this.props.onCloseTimelines}
```

Comment:
Is this just to be more explicit with which props we're passing to the ViewFooter? (Since this file is JSX, I suppose it's for readability purposes)

Reply:
yep. in about 2 PRs, this will all be gone 😄